pwnlib/commandline/template.py

```diff
@@ -22,10 +22,16 @@
 parser.add_argument('--path', help='Remote path of file on SSH server')
 parser.add_argument('--quiet', help='Less verbose template comments', action='store_true')
 parser.add_argument('--color', help='Print the output in color', choices=['never', 'always', 'auto'], default='auto')
+parser.add_argument('--base', help='Path to a custom base template. Defaults to \'~/.config/pwntools/templates/pwnup.mako\'. '
```

Comment:
I think `--template` matches the matter better than `--base`. This isn't a base template, which is extended by other templates, but THE template used to generate the script. Yes `pwn template --template` looks weird, but I can't think of a better name.

The mention of the `.config` template folder should say something like "if it exists"? Right now it sounds like the default template is in the `~/.config` folder while saying the other default template is in yet another path :D